cmd/remove.go

```diff
@@ -34,6 +34,13 @@ var removeCmd = &cobra.Command{
 			log.Error("Error executing command: " + err.Error())
 			os.Exit(1)
 		}
+
+		err = lxclient.SetContainerTemplateRealation(name, "", false)
+		if err != nil {
```

Comment:
Same as above, move the `SetContainerTemplateRelation` function out of the lxclient package